### ISSUE TITLE
Champion Staking

### ISSUE THREAD

**Issue by amsos** [raid-guild/RIPs#87]
Champion Staking is the new process of where we bring in new cohort members into RAID if a new member has a champion who is willing to stake RAID on new members becoming full productive members.

**Comment by amsos** [comment]
Notes copied over from [hack-md notes](https://hackmd.io/IP4s5ec2Sx2GqQ_uIoCfJQ?both):

# Champion Staking Retrospective

* Staking analysis [sheet](https://docs.google.com/spreadsheets/d/1KDsbnIMltT6Ecr2ip0nP4tZHgEd1MwQMVdC2rtheDgc/edit?usp=sharing)

## Champion and Championee reflection convo


--

* Kerp: still has a to-do to request stake back plus reward
    * feels like re-attesting to things people had already done
    * championed people who had already met the criteria before being championed
    * did a weekly meeting with one of them
* Christina: assumed that the cohort stuff would be to prove that you'd be worth championing, and that success criteria were to show that you'd want to stay as a guild member
* Brendan: still murky to signal that he wants to stay
    * was busy when first championed, time is starting to open up now
    * doesn't want to take on something "just to tic a box" if its not the right fit
    * might be challenging to complete a RIP or raid within 60 days
* Kerp: new member shouldn't feel pressure. it's on the champion to ensure that these things happen to get their reward
    * There shouldn't be that much pressure to "make it" in 90 days, this should be clearly labelled in docs
    * Govinda: giving the championed member some of the reward might help motivate both parties
    * criteria were meant to be a representation that the new member was sufficiently/sucessfully engaged 
* Govinda: the existing criteria are a pretty low bar; pretty achievable
    * the 60 days keep up the momentum of the journey 
    * wasn't actually aware of this process
    * might help to segment/questify/phase-ify the new member journey. A little bit more direction on the journey may help
        * Foundations DAO did this, was fun but got hacked
        * eg :point_down: 

![](https://i.imgur.com/czBIol5.jpg)

* Sasquatch: having a third verifier would a) help verify new members are meeting their success criteria, and b) create more opportunities for members to interact with one another
    * champion is still guiding the championee
    * the third person signs off on the raid and other criteria
    * Give a reviewer/vouch for new member
* integration with dungeon master would be valuable
    * Sasquatch: "the champion protocol and integration with dungeon master could also become another product for RaidGuild. As we look to making more products for the space, this kind of onboarding strategy and reputation collecting is ripe for production"

* Profiles
 ![](https://i.imgur.com/xluYeQZ.png)


## Championing goals

* Get more members talking to each other and forming relationships
* Verification: How to ensure Raid Guild's quality and reputation? 
    * Our goals: Bring in great new members and weed out low quality 
    * How to achieve: Create an incentive to bring in these people
* To what degree is the current process successful in supporting these goals? 
    * Christina: There is def a vetting process but that makes being here even better!
    * Gov: Exclusive is good! You don't even need to be a member to do Raids but you want to get in with the cool kids :smile:  
    * When we scale will that have an impact on being a member? Championing process? 
        * Changes information environment, number of connections between members exponentially grows 
        * Putting Raid on the line will stop championing any old 
        

## Retrospective Analysis -- DRAFT

### What happened (since October 26, 2021)
- 21 new members have been championed and joined the guild
    - *compare to 36 new members in the previous ~6 months*
    - 16 were from a cohort (25k RAID), and 5 were "off the street" (50k RAID)
- A total of 650k RAID was staked
    - 300k RAID has been returned
- 10 champions have requested their stake back + reward
    - all have received it; no slashings
    - 250k RAID in rewards has been distributed
- 3 reward requests came prior to the 60 day period
    - 4 reward requests came after 80 days
- 8 champions are eligible to make a request but have not done so
    - 7 over 80 days

[Summary spreadsheet](https://docs.google.com/spreadsheets/d/1KDsbnIMltT6Ecr2ip0nP4tZHgEd1MwQMVdC2rtheDgc/edit?usp=sharing)

### Observations
- some evidence that the staking requirement has reduced the pace of new members
- a number of stake/reward requests were very low quality (missing success criteria reporting) but were voted through anyways
- some members have voiced support for decreasing the reward
- some confusion over the success criteria, such as...
    - can they be completed
    - how strict are they?
    - are they the right measures of the outcomes we want?
- The success criteria may be too low of a bar
- verification is a challenge: the verifier (champion) is the same person that benefits from fudging the verification

### Questions for possible improvement
- should we consider changes to stake amounts?
- should we consider changes to reward amounts?
- should we consider changes to the length of the trial period?
- should we consider changes to success criteria?
- should we consider making membership contingent on completing the success criteria?
- should we consider a more mechanistic protocol?
- how might we better verify the success criteria?


### Next steps
- determine the adjustments to make and a timeframe over which to evaluate them
- make the adjustments
- propose to the guild
- evaluate

**Comment by IanBoucherTru** [comment]
+1 to Spencer's idea of a probationary period after the championee completes the initial success criteria. A role that would enable them to utilize all the tools within the discord while not yet becoming a full member.

**Comment by plor** [comment]
My first impression is that one of the goals of this initiative is to continue bringing in quality members, and as such we should treat this as a market by which the guild rewards this goal being achieved. By market I mean that the highest quality member should receive the highest rewards for championing, and likewise should require the highest stake (via competition to receive such reward). Making objective the aspects by which we agree that a member is of high quality is important both to attract members and to draw the line where candidates don't meet this standard.

I do accept that championing should be a phase between apprentice and member, such that if a championed apprentice doesn't meet the standard, it is easier to move them back to apprentice rather than remove their membership status. The question here is whether a member not meeting the champion metrics should necessarily be removed, or if it is sufficient to slash the champion for failure to onboard successfully.

**Comment by govinda108108** [comment]
Love your work guys, this is great. 

I liked the idea of a third member approving the 

Also keen to integrate lore into the post-kraken slaying part of the journey as they head towards the Guild. "They need a champion ally to enter the walls. The champion will give them a quest and the Guild will approve whether the apprentice has made the cut, (e.g. the gaunlet)".

**Comment by plor** [comment]
Can we formalize this RIP as a `.md` file in the proposals directory of this repo? I think we should close out this issue, so it would be good to have some permanence in a real document to reference. Thoughts on this?
